README.tpm

```diff
@@ -0,0 +1,19 @@
+The following PCRs are extended by shim:
+
+PCR4:
+- the Authenticode hash of the binary being loaded will be extended into
+  PCR4 before SB verification.
+
+PCR7:
+- DBX is extended into PCR7 when we find an entry in it that matches a
+  binary we're trying to launch
+- DB is extended into PCR7 if we find an entry in it that matches a
+  binary we're trying to launch
+- Shim's built-in cert list will be extended into PCR7 if it contains a
```

Comment:
"Shim's vendor cert will be extended into PCR7 if it matches a binary we're trying to launch", and "Shim's build cert will be extended into PCR7 if it matches a binary we're trying to launch"?